README.md

```diff
@@ -224,13 +225,17 @@ docker run -p 80:8080 \
 
 The */where/are/web/files* directory should have a structure like this :
 ```shell
-/where/are/web/files
+/where/are/web/files/
 ├── app1.domain.com
-│   └── index.php
-│   └── ...
+│   ├── app/
+│   │   └── index.php
+│   │   └── ...
+│   └── mysql/
 └── app2.domain.com
-    └── index.php
-    └── ...
+│   ├── app/
+│   │   └── index.php
+│   │   └── ...
+│   └── mysql/
```

Comment:
I think it will be better to do a dedicated section for ROOT_SITE_SUBFOLDER instead of editing the existing one.